services/web/app.py

```diff
@@ -39,7 +39,7 @@ def __prepare_flask_request__(self, request):
         url_data = urlparse(request.url)
         return {
             "https": "on" if request.scheme == "https" else "off",
-            "http_host": request.headers.get("X-Forwarded-Host"),
+            "http_host": request.headers.get("X-Forwarded-Host", ""),
```

Comment:
Wouldn't it be:

`"http_host": request.headers.get("X-Forwarded-Host", request.headers.get("Host", "")),`

?

Reply:
We're conditionally setting `$proxy_host` either to `"X-Forwarded-Host"` or `"Host"` in the NGINX config, which is what populated `"X-Forwarded-Host"`, so it should always have either of those values. Unless I'm missing a case?

Reply:
In either case, behind Nginx, `X-Forwarded-Host` gets set, right? I imagined that this change was for when you're running the policy service not-from-behind Nginx and thus there is no `X-Forwarded-Host`.

Reply:
Ah! Makes sense. I'll add that in, thanks!

Reply:
Updated